scripts/generateFilelistForModule.sh

```diff
@@ -1,114 +1,130 @@
 #!/bin/bash
 
 if [ "$#" -ne 1 ]; then
-    echo "Please provide one and only one module name among [Core|Engine|IO|Gui]"
+    echo "Please provide one and only one module name (e.g. Core) that corresponds to the directory name under src/ (e.g. src/Core)."
     exit 2
 fi
 
 BASE=$1
 LOWBASE=${BASE,,}
-OUTPUT="../cmake/filelist${BASE}.cmake"
+OUTPUT="../src/${BASE}/filelist.cmake"
 
 echo "generate [${BASE}] filelist"
 echo "-- input files from         [../src/${BASE}]"
-echo "-- output vars prefiex with [${LOWBASE}]"
+echo "-- output vars prefix       [${LOWBASE}]"
 echo "-- output file is           [${OUTPUT}]"
 
-rm "${OUTPUT}"
+rm -f "${OUTPUT}"
 
-echo  "# ----------------------------------------------------" >> ${OUTPUT}
-echo  "# ---------- DO NOT EDIT -----------------------------" >> ${OUTPUT}
-echo  "# ----------------------------------------------------" >> ${OUTPUT}
-echo  "# This file is autogenerate from a script:" >> ${OUTPUT}
-echo  "# run form scripts directory" >> ${OUTPUT}
-echo  "# $0 ${BASE}" >> ${OUTPUT}
-echo  "# ----------------------------------------------------" >> ${OUTPUT}
+echo  "# ----------------------------------------------------" >> "${OUTPUT}"
+echo  "# This file can be generated from a script:" >> "${OUTPUT}"
+echo  "# To do so, run form ./scripts directory" >> "${OUTPUT}"
+echo  "# $0 ${BASE}" >> "${OUTPUT}"
```

Comment:
I propose to replace the two lines above by : 
echo  "# To do so, run \"$0 ${BASE}\" from ./scripts directory" >> "${OUTPUT}"